v0/destinations/pinterestConversion/data/pinterestCommonConfig.json

```diff
@@ -0,0 +1,90 @@
+[
+    {
+        "destKey": "event_name",
+        "sourceKeys": "event",
+        "required": false
+    },
+    {
+        "destKey": "event_time",
+        "sourceKeys": "timestamp",
+        "sourceFromGenericMap": true,
+        "required": true,
+        "metadata": {
+            "type": "secondTimestamp"
+        }
+    },
+    {
+        "destKey": "event_source_url",
+        "sourceKeys": "pageUrl",
+        "sourceFromGenericMap": true
+    },
+    {
+        "destKey": "opt_out",
+        "sourceKeys": [
+            "context.device.attStatus",
```

Comment:
opt_out should be set to true or false based on the status of this property
context.device.attStatus when the value is 3 we should send opt_out as true for all other values it should be false

We need to implement this custom logic in case of iOS payload

Reply:
added